lit/setup-and-operations/install.lit

```diff
@@ -68,6 +68,232 @@ page.
   different machine than the server.
 }
 
+
+\section{
+  \title{Configuring Auth Providers}{auth-config}
+
+  At startup, you will need to configure any local users or third-party auth
+  providers that you wish to use with Concourse.
+
+  In contrast to previous releases of Concourse, \code{users} are now central
+  to the authentication flow. Instead of logging in as a team, you now log in
+  as a user and can belong to one or more teams. Users can be added to a team
+  by configuring the team's whitelist as described in
+  \reference{authentication}.
+
+  \section{
+    \title{Local Auth}{local-auth-config}
+
+    In order to add new users to the system, you can supply the
+    \code{--add-local-user} flag to the \code{concourse web} subcommand like
+    so:
+
+    \codeblock{bash}{{{
+    concourse web \
+      --add-local-user myuser:mypass
+    }}}
+  }
+
+  \section{
+    \title{GitHub Auth}{github-auth-config}
+
+    A Concourse server can authenticate against GitHub to take advantage of
+    their permission model and other security improvements in their
+    infrastructure.
+
+    \section{
+      \title{Creating a GitHub application}
+
+      First you need to \link{create an OAuth application on
+      GitHub}{https://github.com/settings/applications/new}.
+
+      The callback URL will be the external URL of your Concourse server with
+      \code{/sky/issuer/callback} appended. For example, Concourse's own CI
```

Comment:
Is this right? Ours is `https://ci.concourse-ci.org/sky/dex/callback/github`. (I think the `/github` is redundant, but ours says `dex` not `issuer`.)

Reply:
We're trying to remove all the outward references to `dex`. Figured `issuer` was a nice generic name.